.all-contributorsrc

```diff
@@ -40,6 +40,17 @@
       "contributions": [
         "code"
       ]
+    },
```

Comment:
Rollback this change. This is not done manually but rather by command. Google it to understand